.storybook/components/checkered-background/CheckeredBackground.css

```diff
@@ -1,7 +1,7 @@
 .o-ui-sb-checkered-background-preview {
```

Comment:
When opening the collapsible menu by clicking it and then pressing the down arrow key, it focuses the second option instead of the first one:
![image](https://github.com/gsoft-inc/wl-orbiter/assets/110498164/4ee51614-8888-465f-b524-6a2b6c49fb36)

If you open the menu using the keyboard, it works as expected.

Reply:
Good catch, it's the same in Orbit therefore we should open an issue in Orbit and once fixed, fix it here too. I would not block a release for this especially as it's a trivial/edge case.